core/server/src/main/java/alluxio/worker/block/BlockMetadataManagerView.java

```diff
@@ -229,4 +229,15 @@ public BlockMeta getBlockMeta(long blockId) throws BlockDoesNotExistException {
       return null;
     }
   }
+
+  /**
+   * Return the byte size of the block.
+   *
+   * @param blockId id of the block
+   * @return block size
+   * @throws BlockDoesNotExistException if no {@link BlockMeta} for this block id is found
+   */
+  public long getBlockSizeIfExist(long blockId) throws BlockDoesNotExistException {
```

Comment:
@uronce-cc I think it's not like `mMetadataManagerView.getBlockMeta()` called when evicting blocks. We can choose to evict other blocks if the current block return a null. But if we fail to get the accurate size of blocks in `onCommit()` method, LIRS may fail to know how much space LIR cache and HIR cache have used.
 
I will remove `getBlockSizeIfExist()` method and if `mMetadatgaManagerView.getBlockMeta().getBlockSize()` returns null, throws BlockDoesNotExistException() to guarantee LIRS can run correctly.